Ports/qoi/patches/0001-Add-stb_image-headers.patch

```diff
@@ -0,0 +1,9647 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Liav A <liavalb@gmail.com>
+Date: Sun, 26 Jun 2022 22:11:12 +0300
+Subject: [PATCH] Add stb_image headers
```

Comment:
This should really just be a separate port. I don't see much reason in "vendoring" it.